File: shells/fish/files/patch-util.c

--- util.c.orig	2010-01-13 06:44:46.000000000 -0800
+++ util.c	2010-01-13 06:46:39.000000000 -0800
@@ -320,7 +320,7 @@
 {
 	int pos;
 	
-	if( (float)(h->count+1)/h->size > 0.75f )
+	if( (float)(h->count+1) > 0.75f * h->size )
 	{
 		if( !hash_realloc( h, (h->size+1) * 2 -1 ) )
 		{

File: dns/dns_balance/files/patch-util.rb
--- util.rb.orig	Fri Dec 22 12:46:39 2000
+++ util.rb	Mon Mar  5 13:53:57 2001
@@ -27,7 +27,7 @@
 #
 # ex) "127.0.0.1" => "\x7f\x0\x0\x1"
 def str_to_ipstr(str)
-  arr = str.split(".").filter{|i| i.to_i}
+  arr = str.split(".").collect{|i| i.to_i}
 
   if arr.length != 4
     put_log("ipaddr invalid\n")
